distil/primitives/isolation_forest.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+import math
+from typing import List, Optional
+import typing
+
+import numpy as np
+import pandas as pd
+
+from d3m import container, utils
+from d3m.metadata import base as metadata_base, hyperparams, params
+from d3m.primitive_interfaces import base, unsupervised_learning
+
+from sklearn.ensemble import IsolationForest
+
+from distil.utils import CYTHON_DEP
+import version
+
+__all__ = ("PrefeaturisedPoolingPrimitive",)
+
+logger = logging.getLogger(__name__)
+
+
+class Hyperparams(hyperparams.Hyperparams):
+    n_jobs = hyperparams.Hyperparameter[int](
```

Comment:
should have lower and upper bounds defined for int/float parameters

Reply:
for n_jobs specifically, I don't think any of the n_job hyperparams have an upper/lower bound